src/app/item-popup/PlugTooltip.tsx

```diff
@@ -62,16 +63,26 @@ export default function PlugTooltip({
       {defs && Boolean(plug?.plugDef?.investmentStats?.length) && (
         <div className="plug-stats">
           {plug.stats &&
-            _.sortBy(Object.keys(plug.stats), (h) =>
-              statAllowList.indexOf(parseInt(h, 10))
-            ).map((statHash) => (
-              <StatValue
-                key={statHash + '_'}
-                statHash={parseInt(statHash, 10)}
-                value={plug.stats![statHash]}
-                defs={defs}
-              />
-            ))}
+            _.sortBy(Object.keys(plug.stats), (h) => statAllowList.indexOf(parseInt(h, 10)))
+              .filter((statHash) =>
+                isPlugStatActive(
+                  item,
+                  plug.plugDef.hash,
+                  Number(statHash),
+                  Boolean(
+                    plug.plugDef.investmentStats.find((s) => s.statTypeHash === Number(statHash))
+                      ?.isConditionallyActive
+                  )
+                )
+              )
+              .map((statHash) => (
+                <StatValue
+                  key={statHash + '_'}
```

Comment:
Do you know why this key has `_` appended to it?

Reply:
No Idea, that was pre-existing.